gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/EhrExtractMessageHandler.java

```diff
@@ -41,38 +49,88 @@ public void handleMessage(InboundMessage inboundMessage, String conversationId)
         );
     }
 
-    //TODO: this method is related to the large messaging epic and should be called after saving translated Boundle resource.
-    //  Can be used during implementation of NIAD-2045
-    private boolean sendContinueRequest(RCMRIN030000UK06Message payload, String conversationId, String patientNhsNumber) {
-        // TODO: Should call
-        //  sendContinueRequestHandler.prepareAndSendRequest(prepareContinueRequestData(payload, conversationId, patientNhsNumber));
-        return true;
+    boolean sendNackMessage(NACKReason reason, RCMRIN030000UK06Message payload, String conservationId) {
```

Comment:
This method is not used anywhere. I thought this ticket was about general error handling - so any errors thrown in the `handleMessage` method would be caught and then this method would be called. Right now, the error statuses only cover the large messaging issues, could you add maybe at least one general error status?

Reply:
I've added acknowledgement codes for `CLINICAL_SYSTEM_INTEGRATION_FAILURE`, `EHR_EXTRACT_CANNOT_BE_PROCESSED` and `UNEXPECTED_CONDITION`. The last one is a general code, and the other two cover the scenarios covered in [handling acknowledgements](https://gpitbjss.atlassian.net/wiki/spaces/NIA/pages/11747655681/Handling+Acknowledgments).

Reply:
I've mapped all three of these messages to a migration status of `EHR_GENERAL_PROCESSING_ERROR`.